src/Stratis.Bitcoin.Features.RPC/Controllers/FullNodeController.cs

```diff
@@ -205,36 +205,36 @@ public BlockHeaderModel GetBlockHeader(string hash, bool isJsonFormat = true)
         /// Returns information about a bitcoin address
         /// </summary>
         /// <param name="address">bech32 or base58 BitcoinAddress to validate.</param>
-        /// <returns>JObject containing a boolean indicating address validity</returns>
+        /// <returns>ValidatedAddress containing a boolean indicating address validity</returns>
```

Comment:
Change this to: 

`/// <returns><see cref="ValidatedAddress"/> containing a boolean indicating address validity.</returns>`